DESCRIPTION

```diff
@@ -54,7 +54,10 @@ Imports:
     magrittr,
     flipr,
     cli,
-    withr
+    withr,
+    progress,
```

Comment:
Where do you use the `progress` package?

Reply:
As commented before:
> I have updated DESCRIPTION file importing `progressr` and `furrr`. However, checks report:
> 
> ```
> Error: Error in loadNamespace(x) : there is no package called 'progress'
> ```
> 
> I do not explicitly use `progress`, should I add this package to "Imports" in DESCRIPTION file?

I solve errors by adding the `progress` package in DESCRIPTION file, but I do not use it explicitly.